install/install.php

```diff
@@ -306,29 +306,35 @@ function pluginGlpiinventoryInstall($version, $migrationname = 'Migration')
     require_once(PLUGIN_GLPI_INVENTORY_DIR . "/inc/inventorycomputerstat.class.php");
     PluginGlpiinventoryInventoryComputerStat::init();
 
-   /*
-    * Define when install agent_base_url in entity
-    */
-    $full_url = filter_input(INPUT_SERVER, "PHP_SELF");
-    $https = filter_input(INPUT_SERVER, "HTTPS");
-    $http_host = filter_input(INPUT_SERVER, "HTTP_HOST");
-    if (!empty($full_url) && !strstr($full_url, 'cli_install.php')) {
```

Comment:
Can you explain why you removed the `!strstr($full_url, 'cli_install.php')` check ?